### PR TITLE
Ntrnl 240 update api sdk module

### PR DESCRIPTION
**Pull request by mel-am**
### JIRA link

Resolves[ NTRNL-240 ](https://technologyprogramme.atlassian.net/browse/NTRNL-240)
### Description

Updated api-sdk

### Work checklist

- [ ] Tests added where applicable
- [ ] No vulnerability added

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,14 +1,22 @@
 {
-  "name": "node_sdk_prototype",
+  "name": "@co-digital/node_api_sdk",
```
Comment (Mouhajer-CO) [inline package.json:2]: I think a better name is `@co-digital/api_sdk`

File: package.json [hunk 2]
```diff
@@ -27,5 +35,9 @@
   },
   "dependencies": {
     "axios": "^1.6.0"
-  }
+  
+},
+"files": [
+  "./lib/**/*"
+]
```
Comment (Mouhajer-CO) [inline package.json:42]: Add `keywords` field  with:
```
{
   ...,
  "keywords": [ "api",  "sdk", "axios" ]
}
```

File: package.json [hunk 3]
```diff
@@ -1,14 +1,22 @@
 {
-  "name": "node_sdk_prototype",
+  "name": "@co-digital/node_api_sdk",
   "version": "1.0.0",
-  "description": "Node SDK Prototype",
+  "description": "A Node api sdk Node.JS applications in CO Digital.",
+  "homepage": "https://github.com/cabinetoffice/node-api-sdk#README.md",
```
Comment (Mouhajer-CO) [inline package.json:5]: Add `types` field

Comment (mel-am) [reply package.json:5]: "repository": {
    "type": "git",
    "url": "https://github.com/cabinetoffice/api-sdk.git"
  },

This is within the package already - is this ok?

File: package.json [hunk 4]
```diff
@@ -27,5 +37,17 @@
   },
   "dependencies": {
     "axios": "^1.6.0"
-  }
+
```
Comment (Mouhajer-CO) [inline package.json:40]: Could you please check the format, thanks

**Review by Mouhajer-CO — APPROVED** [review]

**Pull request merged**